library/libraries.js

```diff
@@ -1862,5 +1862,17 @@ var d41d8cd98f00b204e9800998ecf8427e_LibraryDetectorTests = {
             
             return false;
         }
+    },
+    'Bento': {
+        id: 'bentojs',
+        icon: 'bentojs',
+        url: 'https://bentojs.dev',
+        npm: 'https://www.npmjs.com/org/bentoproject',
+        test: function (win) {
+          if (win.BENTO === undefined) {
```

Comment:
Is there an additional property that can also be checked for to minimize any potential false positives? E.g. Can we do `win.BENTO && win.BENTO.push`?

Reply:
Sure! Done!